### DISCUSSION TITLE
Innteractive Map

### DISCUSSION THREAD

**Discussion by jcjp** [unovue/inspira-ui#34] [Ideas]
It would be a good alternative to Globe to have a simple interactive map or just a simple map where we could plot points or color a certain region. Great library by the way!

**Comment by nathan-de-pachtere** [comment]
@jcjp Do you have an existing live example ? Screenshot  ?

**Reply by jcjp** [reply]
I think what I saw where pretty complex, but a simple polygon map similar to the Globe would do, instead of a 3d round just a flat map.

I saw different implementations you can get inspiration from the libraries like am5charts, leaflet and openlayers.

**Comment by nathan-de-pachtere** [comment]
I see. What do you expect Inspira UI to add on top of a map or chart library? You can already integrate all JS map and chart libraries within Vue.js.

**Reply by jcjp** [reply]
Just an alternative option, it could be the same to Globe but on a flat surface instead of 3d, like this:

![Map](https://github.com/user-attachments/assets/eb906007-6bd3-4888-8003-14970d4ab14b)

And then I guess whatever you want to add as a scope in it, like add a marker or animations but for my side a simple map like that would do.

**Comment by rahul-vashishtha** [comment]
@jcjp Did you mean something like this? https://melonman0.github.io/InteractiveMap/

**Reply by jcjp** [reply]
> @jcjp Did you mean something like this? https://melonman0.github.io/InteractiveMap/

Yes! I think the only thing left is to have an country pre-selected and highlighted if it's easy to do.
